tests/baselines/reference/computedEnumTypeWidening.js

```diff
@@ -52,6 +52,16 @@ function f4() {
     let v1 = E.B;  // E2
 }
 
+const c1 = E.B;
+const c2 = E.B as const;
+let v1 = E.B;
+let v2 = E.B as const;
+
+class C {
+  p1 = E.B;
+  p2 = E.B as const;
```

Comment:
A `readonly p3 = E.B` would be another good addition, since that one should be like a `const` declaration, iirc.